package.json

```diff
@@ -1,5 +1,12 @@
 {
+  "scripts": {
+    "start": "node bms-monitor.js",
+    "test": "npx mocha"
+  },
   "devDependencies": {
     "chai": "^4.3.6"
+  },
+  "dependencies": {
+    "mocha": "^10.3.0"
```

Comment:
`mocha` is a development-time dependency like `chai`. We don't want it to go with the product, so let it be together with chai in `devDependencies`